---
title: "PackageInstructions"
output: html_document
---

## Installing ansGreatImagesSometimes:
## from github: 

```{r}

library(devtools)
devtools::install_github("andrewsherman94/Project3")

```

 -----------------------------------------------------

## For seq_req()
## Function to query sequences from genbank given a search term and gene

## Requires 3 arguments:
## 1. target -- the search term for genbank -- can be species, genus, family, etc.
## 2. gene -- the gene you'd like to search for -- such as cytb, COI, RAG1, etc.
## 3. return_max -- the number of reads to return 

## Requires the packages rentrez 


```{r}

seq_req(target = "Characodon", gene = "cytb", return_max = 15)

```

## Note that the target and gene need to be in quotes, return_max does not.
## The output .fasta can be exported and used in tree construction or other analyses

 --------------------------------------------------------
 
## For plot_points()
## Function to plot  GPS coordinates in a .csv file to a map of Louisiana -- for rivers.

## Requires a read-in .csv file saved as an object in decimal degrees (not DMS)
## -- example: rivers_la.csv in "/cloud/project/"

## Requires the packages sp, rgdal, tidyverse, dplyr

```{r}

rivers <- read_csv("/cloud/project/rivers_la.csv")

```

## Function requires 4 arguments:
## 1. coords -- the name of the read-in .csv file object
## 2. site_name -- the name of the column in your .csv that contains site/river names
## 3. longitude -- the name of the column in your .csv that containts longitude coords
## 4. latitude -- the name of the column in your .csv that containts latitude coords

## example:
```{r}

plot_points(coords = rivers, site_name = "Site", longitude = "Long", latitude = "Lat")

```
## Note that site_name needs to be in quotes, but the coords object does not.


## If the results are adequate and map properly, then the output can be saved:

```{r}

png("filename.png")
plot_points(coords = rivers, site_name = "Sites", longitude = "Long", latitude = "Lat")
dev.off()

```
